py/py_sparkling/ml/models.py

```diff
@@ -66,9 +67,18 @@ def predict(self, dataframe):
         return self.transform(dataframe)
 
     def get_input_names(self):
+        warnings.warn("The method 'get_input_names' is deprecated. Use 'get_features_colssss' instead!")
         return list(self._java_obj.getInputNames())
 
+    def get_features_cols(self):
```

Comment:
Naming conventions of this class are inconsistent with other classes since snake case used here.

Reply:
Can we deprecated the snake case and recommend the camel case?

Such as:
```
warnings.warn("The method 'get_input_names' is deprecated. Use 'getFeaturesCols' instead!")
```

We should use camel case here as well